hub/management/commands/import_mps.py

```diff
@@ -41,6 +50,7 @@ def handle(self, quiet: bool = False, *args, **options):
         self._quiet = quiet
         self.import_mps()
         self.import_mp_images()
+        self.check_for_duplicate_mps()
```

Comment:
I might put this ahead of `import_mp_images` as then it's a) not fetching images for the old MP and b) the fetch bit takes a while so it reduces the time where the "two records found" error potentially appears.